package.json

```diff
@@ -32,6 +32,7 @@
     "connect-multiparty": "^2.2.0",
     "cookie-parser": "^1.4.3",
     "cookie-sessions": "github:auth0/cookie-sessions#53a8aae",
+    "diff_match_patch": "^0.1.1",
```

Comment:
Are we using it anywhere?

Reply:
Yes, it's used but was undeclared, that's why I added it.

Reply:
Used here: https://github.com/auth0/ad-ldap-connector/blob/0d3fda67dcfafd60de9feeced1d994913dffdd67/admin/public/codemirror-addon/merge/merge.js#L8